cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html

```diff
@@ -46,8 +46,8 @@ <h2 class="header-slug">
     </div>
 
     <div class="o-sidebar-contact-info_content">
-        {% if value.header %}
-            <h3>{{ value.header }}</h3>
+        {% if value.contact.heading %}
+            <h3>{{ value.contact.heading }}</h3>
```

Comment:
Minor thing: the documentation above in this file says `contact.header` versus `contact.heading`. Should the documentation be updated?